src/AppMain.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;700;800&display=swap');
+
+div {
+  background: #EEF0F1;
+}
+
+.App {
+  text-align: center;
+  font-family: 'Mulish', sans-serif;
+  min-height: 50rem;
+}
+
+.NavBar {
+  width: 100%;
+  height: 5rem;
+  text-align: center;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  flex-direction: row;
+  position: fixed;
+  top: 0;
+  background: #4267B2;
+  color: white;
+  min-width: 50%;
+  white-space: nowrap;
+  font-weight: 800;
+  z-index: 1;
+}
+
+#TeachLA {
+  width: 3.75rem;
+  height: 3.75rem;
+  left: 20px;
+  margin: 1.25rem;
+}
+
+#BiasByUs {
+  font-size: 3rem;
+  margin: 0.5625rem;
+}
+
+#currentCaseStudy {
+  font-size: 1.875rem;
+  margin: 1.25rem;
+  position: relative;
+  top: 0.3125rem;
+}
+
+button {
+  background-color: #EEF0F1;
+  width: 4.063rem;
+  min-width: 4.063rem;
+  height: 4.063rem;
+  border-radius: 50%;
+  border: none;
+  position: relative;
+  margin-left: auto;
+  margin-right: 4.375rem;
+  display: inline-block;
+}
+button:focus {
+  outline: none;
+}
+.triangle {
+  width: 0;
+	height: 0;
+	border-left: 1.25rem solid transparent;
+	border-right: 1.25rem solid transparent;
```

Comment:
Indentation?

Also (but not merge-blocking), this may just be easier with an SVG than fiddling around with an empty DOM element. Will be better for accessibility as well, since you can use `aria-label`.

Reply:
Oh this was for creating a triangle...I found code online and tweaked it to the shape I wanted. Do you have any suggestions for making a proper triangle through css? Or should I get a picture/SVG of a triangle and edit that?

Reply:
Not a worry for now. I think this is the right (if slightly-janky) way to do it with CSS, but normally I think SVGs are the way to go for performance/accessibility. A triangle SVG is pretty simple to make (it's just 3 points), so you can find one online or make it yourself. We can worry about this later though!